build/ga_build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+
+set -o pipefail
+
+# Master builds release to Beta
+yarn run publish-beta
+VERSION=$(node -p -e "require('./package.json').version + '.' + process.env.GITHUB_RUN_NUMBER")
```

Comment:
This'll likely collide with the Travis numbers?

Reply:
Now I am adding 1,000,000 to all the build numbers coming from github. It would be nice to remove this eventually as it needs to be kept inline in 3 different area's as you will see.